tests/foreman/cli/test_contentview.py

```diff
@@ -5302,7 +5302,6 @@ def test_positive_arbitrary_file_sync_over_capsule(self):
         :CaseLevel: System
         """
 
-    @stubbed()
     @tier2
     def test_positive_arbitrary_file_repo_promotion(self):
         """Check arbitrary files availability on Environment after Content
```

Comment:
The docstring is complete? 'after content' what? do you mean content view promotion?

Reply:
Yes, Here we are validating the arbitrary file availability after content- view version promotion.